how-to-be-critical.html

```diff
@@ -62,7 +62,16 @@
 			<li>Me: I wouldn't go that far. Some of that metadata like who else received the message, could be pretty important. Maybe try putting the names of the other recipients.</li>
 		</ul>
 
-		<p>There are several things to notice about the exchange above. It's respectful, with each person listening and accepting what the other person is saying. It's collaborative, with each person contributing knowledge to the conversation. It's grounded in design rationale and design judgement, focusing on <em>why</em> choices are made and why <em>different</em> choices were made, and how that might effect the success of the solution.</p>
+		<p>There are several things to notice about the exchange above. It's respectful, with each person listening and accepting what the other person is saying. It's collaborative, with each person contributing knowledge to the conversation. It's grounded in design rationale and design judgement, focusing on <em>why</em> choices are made and why <em>different</em> choices were made, and how that might effect the success of the solution.</p> 
+
+		<p>Another form of critique that can be applied to design is <a href=""></a>Socratic questioning. In this form of critique, the person giving the critique wants to deeply probe the designer's way of thinking and dig beneath the surface of their design. Some types of questions to achieve this include:</p>
```

Comment:
There's a missing link here. I'm not sure what to link to; maybe you can find a resource? If not, just remove the link.

Reply:
Just added it